inference-engine/tests/functional/plugin/shared/src/behavior/stress_tests.cpp

```diff
@@ -38,12 +38,15 @@ TEST_P(MultipleAllocations, InferWorksCorrectAfterAllocations) {
 
     // Experiments demonstrated that 10 cycles are enough to reproduce the issue
     int infersCount = 10;
+    auto isDataGenerated = 0;
+
     for (int j = 0; j < infersCount; ++j) {
         LoadNetwork();
 
         std::cout << "Infer(): " << j << std::flush;
-
-        GenerateInputs();
+        if (!isDataGenerated++) {
```

Comment:
`if (j == 0) ...`
Not the best solution also, but shorter

Reply:
Changed.